### DISCUSSION TITLE
user name blacklisted: False

### DISCUSSION THREAD

**Discussion by f242** [DigitalRuby/IPBan#186] [Q&A] [answered]
part from log:
IP blacklisted: False, **_user name blacklisted: False_**, fails user name white list regex: False, user name edit distance blacklisted: False
can you describe `user name blacklisted`?
i thought this is something like:
`<add key="UserNameBlacklist" value="ADMINISTRATOR,ADMIN,USER"/>`
but it  doesn't work and i din't find an answer in documentation

**Comment by jjxtra** [comment]
All config is spelled out here: https://github.com/DigitalRuby/IPBan/wiki/Configuration

**Comment by f242** [comment]
blacklist only for dns or ip
for username i see only whitelist

**Comment by jjxtra** [comment, marked as answer]
Correct. If you want to whitelist your own user names in the user name whitelist feel free, then the first failed login for a user name not in the whitelist will get banned.

**Reply by f242** [reply]
it still looks very strange, there is no balacklist setting where you could specify the most common names for username, but in the log it is present

**Comment by jjxtra** [comment]
You can put user names in the blacklist property but it’s largely not
needed if you just whitelist your own user name, which has the effect of
blacklisting every other user name.

On Tue, Apr 5, 2022 at 1:56 AM f242 ***@***.***> wrote:

> it still looks very strange, there is no balacklist setting where you
> could specify the most common names for username, but in the log it is
> present
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/DigitalRuby/IPBan/discussions/186#discussioncomment-2506165>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AAJWQGAG3JJHD34FT6PZ76LVDPWZTANCNFSM5SREHF6A>
> .
> You are receiving this because you commented.Message ID:
> ***@***.***>
>
-- 
-- Jeff
